doc/data/glossary.yml

```diff
@@ -12,223 +12,229 @@
   description: LoRaWAN can use channels with a bandwidth of either 125 kHz, 250 kHz or 500 kHz, depending on the region or the Frequency Plan. Making the bandwidth 2x wider (from BW125 to BW250) allows you to send 2x more bytes in the same time.
   link: https://www.thethingsnetwork.org/docs/lorawan/modulation-data-rate.html
 
-Spreading Factor:
+- term: Spreading Factor
   description: The transmission speed or Data Rate of a LoRaWAN message, ranging from SF7 (highest Data Rate) to SF12 (lowest Data Rate). Making the spreading factor 1 step lower (from SF10 to SF9) allows you to roughly send the same amount of data use half the time on air. Lowering the spreading factor makes it more difficult for the gateway to receive a transmission, as it will be more sensitive to noise.
   link: https://www.thethingsnetwork.org/docs/lorawan/modulation-data-rate.html
   abbr:
     - SF
 
-Adaptive Data Rate:
+- term: Adaptive Data Rate
   description: A mechanism for optimizing data rates, airtime and energy consumption in the network.
   link: https://www.thethingsnetwork.org/docs/lorawan/adaptive-data-rate.html
   abbr:
     - ADR
 
-Signal to noise ratio:
+- term: Signal to noise ratio
   description: A measure of signal strength.
   link: https://en.wikipedia.org/wiki/Signal-to-noise_ratio
   abbr:
     - SNR
 
-Classes:
+- term: Classes
   description: The LoRaWAN specification defines three device types. All LoRaWAN devices must implement Class A, whereas Class B and Class C are extensions to the specification of Class A devices.
   link: https://www.thethingsnetwork.org/docs/lorawan/classes.html
 
-Class A:
+- term: Class A
   description: Class A devices support bi-directional communication between a device and a gateway. Uplink messages (from the device to the server) can be sent at any time (randomly). The device then opens two receive windows at specified times after an uplink transmission. The first receive window (Rx1) is set to 5 seconds and can be modified. The second receive window (Rx2) always comes 1 second after Rx1.
   link: https://www.thethingsnetwork.org/docs/lorawan/classes.html
 
-Class B:
+- term: Class B
   description: Class B devices extend Class A by adding scheduled receive windows for downlink messages from the server. Using time-synchronized beacons transmitted by the gateway, the devices periodically open receive windows.
   link: https://www.thethingsnetwork.org/docs/lorawan/classes.html
 
-Class C:
+- term: Class C
   description: Class C devices extend Class A by keeping the receive windows open unless they are transmitting. This allows for low-latency communication but is many times more energy consuming than Class A devices.
   link: https://www.thethingsnetwork.org/docs/lorawan/classes.html
 
-Device ID:
-  description: A unique, human-readable identifier for your device.
+- term: Device ID
+  description: A unique, human-readable identifier for your device. You make it up, so be creative. Device IDs can not be reused.
```

Comment:
Device IDs can not be used by multiple devices in the same application. They can be re-used across different applications